internal/services/containers/kubernetes_cluster_node_pool_resource.go

```diff
@@ -81,6 +81,9 @@ func resourceKubernetesClusterNodePool() *pluginsdk.Resource {
 			pluginsdk.ForceNewIfChange("upgrade_settings.0.drain_timeout_in_minutes", func(ctx context.Context, old, new, meta interface{}) bool {
 				return old != 0 && new == 0
 			}),
+			pluginsdk.ForceNewIfChange("upgrade_settings.0.undrainable_node_behavior", func(ctx context.Context, old, new, meta interface{}) bool {
```

Comment:
can we add a note in the docs to explain that once added, this property cannot be removed? Also, could you check if this is also applicable to this property in kubernetes_cluster?